config/jobs/kubernetes-sigs/cluster-api-boostrap-provider-kubeadm/cluster-api-bootstrap-provider-kubeadm-presubmits.yaml

```diff
@@ -4,12 +4,15 @@ presubmits:
   - name: pull-cluster-api-bootstrap-provider-kubeadm-verify
     path_alias: "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm"
     always_run: true
+    optional: false
     decorate: true
     spec:
       containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20190805-63855bd-master
         command:
         - "./hack/verify-all.sh"
+        securityContext:
+          privileged: true
```

Comment:
the config also needs the dind preset, see:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml#L8

also the command has to change to:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml#L32-L34

(needs to have `runner.sh` first which setups dind on runtime.

Reply:
Will fix, thanks.